### PR TITLE
Adding .NET Core to overview

### PR DESCRIPTION
**Pull request by sourabhkatti**
Added .NET Core as a language
Renamed ".NET agent" to ".NET Framework agent"

### CODE REVIEW

File: content/installation/setup/Installation-quick-start.md [hunk 1]
```diff
@@ -34,7 +34,9 @@ To connect to Contrast agents, install an agent into your web application server
 
 * The [Java agent](installation-java.html#java-overview) analyzes the behavior of Java web applications running on your container of choice. 
 
-* The [.NET agent](installation-netinstall.html) analyzes the behavior of .NET web applications running on IIS as users interact with these applications.
+* The [.NET Framework agent](installation-netinstall.html) analyzes the behavior of .NET web applications running on IIS as users interact with these applications.
+
+* The [.NET Core agent](installation-netcore.html#netcore-overview) instruments and analyzers applications running in the .NET Core runtime.
```
Comment (kaitlynmortimer) [inline content/installation/setup/Installation-quick-start.md]: @sourabhkatti 

Typo: "instruments and **analyzers** applications"

**Comment by Silvenga** [comment]
@sourabhkatti do we still want to merge this?

**Comment by Silvenga** [comment]
Poke @sourabhkatti

**Comment by sourabhkatti** [comment]
@kaitlynmortimer @Silvenga Just made the fix, is this one ready to push out?

**Pull request merged**